src/main/kotlin/uk/gov/justice/digital/hmpps/hmppschallengesupportinterventionplanapi/domain/DecisionAndActionsTranslation.kt

```diff
@@ -0,0 +1,72 @@
+package uk.gov.justice.digital.hmpps.hmppschallengesupportinterventionplanapi.domain
+
+import uk.gov.justice.digital.hmpps.hmppschallengesupportinterventionplanapi.entity.DecisionAndActions
+import uk.gov.justice.digital.hmpps.hmppschallengesupportinterventionplanapi.entity.ReferenceData
+import uk.gov.justice.digital.hmpps.hmppschallengesupportinterventionplanapi.entity.Referral
+import uk.gov.justice.digital.hmpps.hmppschallengesupportinterventionplanapi.enumeration.Reason
+import uk.gov.justice.digital.hmpps.hmppschallengesupportinterventionplanapi.enumeration.Source
+import uk.gov.justice.digital.hmpps.hmppschallengesupportinterventionplanapi.model.request.CreateDecisionAndActionsRequest
+import java.time.LocalDate
+import java.time.LocalDateTime
+
+fun DecisionAndActions.toModel() =
+  uk.gov.justice.digital.hmpps.hmppschallengesupportinterventionplanapi.model.DecisionAndActions(
```

Comment:
Best to use `import uk.gov.justice.digital.hmpps.hmppschallengesupportinterventionplanapi.model.DecisionAndActions as DecisionAndActionsModel` then reference `DecisionAndActionsModel` here

Reply:
I've got two imports with the same name - Bill has had to do the same thing with his `SaferCustodyScreeningOutcomeTranslation` :)

Reply:
Yep and that's why you do the `as DecisionAndActionsModel` so it's not called the same as the entity ;)

I missed it with Bill's PR. You can change his as the screenshot below
![Uploading image.png…]()

Reply:
Fixed by deleting the whole thing and moving the one function left that's needed as we discussed :)